src/data.toml

```diff
@@ -9,7 +9,7 @@ fix_voice_activity = "The cause of this is unclear. To fix it, please disable `A
 no_mod_settings = "Try visiting [this link](<https://raw.githubusercontent.com/pyoncord/detta-builds/main/bunny.js>) and see if it loads bunch of texts. If it doesn't, try using a VPN and/or disable any sort of ad blocker and try again.\nThese troubleshooting steps may still resolve your issue even if the link loads a bunch of text."
 custom_fonts = "Fonts are experimental and some packs may break the application. You can get font packs listed here: https://discord.com/channels/1196075698301968455/1225584192012947506/1241453802184380418"
 ios_notifications = "For notifications to work on a Jailed device you need to sign the ipa with a certificate that has the notification entitlement enabled.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system.\nYou may use a system information app such as `CPU-Z` or `AIDA64.` to check the architecture of your CPU and operating system."
+thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system.\nYou may use a system information app such as `CPU-Z` or `AIDA64`. to check the architecture of your CPU and operating system."
```

Comment:
moving the period still breaks this grammatically, there shouldn't be any period, it's a single statement

Reply:
done, pushing changes